### PR TITLE
Improving --noprogressbar handling behaviour

### PR DESCRIPTION
**Pull request by AstroSnail**
Once again, but now I fixed an enormous problem I somehow didn't see the first time coming.

### CODE REVIEW

File: src/lib/pacman.sh.in [hunk 1]
```diff
@@ -56,7 +56,8 @@ pacman_cmd() {
 pacman_refresh() {
 	local _arg
 	title $(gettext 'synchronizing package databases')
-	(( $1 > 1 )) && _arg="-Syy" || _arg="-Sy"
+ 	(( $1 > 1 )) && _arg="-Syy" || _arg="-Sy"
```
Comment (larchunix) [inline src/lib/pacman.sh.in]: From `git-apply`: `space before tab in indent.`

File: src/lib/pacman.sh.in [hunk 2]
```diff
@@ -56,7 +56,8 @@ pacman_cmd() {
...
+ 	(( $HIDEPROGRESS )) && _arg="$_arg --noprogressbar"
```
Comment (larchunix) [inline src/lib/pacman.sh.in]: From `git-apply`: `space before tab in indent.`

**Comment by AstroSnail** [comment]
I believe I have fixed the issues you mentioned, and the commits have been squashed

**Comment by larchunix** [comment]
Could you change the commit message `Squashing commits` to describe what the commit is about ?

**Comment by AstroSnail** [comment]
done

**Comment by larchunix** [comment]
Thanks, lets merge it.

**Pull request merged**